frontend/src/features/utils/views/ComponentsView.tsx

```diff
@@ -20,30 +65,16 @@ const ComponentsView = () => (
       <br />
       <Button>Główny przycisk</Button>
     </Item>
-
     <Item>
       <br />
       Header component
       <Header />
     </Item>
-
     <Item>
       <br />
-      Content component
-      <Content>1</Content>
+      Table component
+      <Table pagination={{ pageSize: 20 }} columns={columns} dataSource={dataSource} />
```

Comment:
To byśmy zrobili troszkę inaczej. Domyślnie niech każdy element `<Table />` posiada ilość elementów równą 20 (20 wydzielone do stałej, np. sizes).

Dzięki temu na każdym ekranie będziemy mieli tą samą ilość danych.

Wyglądać to mniej więcej będzie tak w pliku `Table.tsx`;

const StyledTable = styled(ATable)`...`;

const Table: React.VFC<TableProps> = () => (
    <StyledTable pagination={{ pageSize: sizes.list }}  />
);

export default Table;